src/cloudkit.cs

```diff
@@ -277,15 +480,15 @@ interface CKFetchNotificationChangesOperation {
 		[NullAllowed] // by default this property is null
 		[Export ("notificationChangedBlock", ArgumentSemantic.Copy)]
 		Action<CKNotification> NotificationChanged {
-			[NotImplemented ("Only setting the handler is currently supported.")]
+			[iOS (10,0), Watch (3,0), TV (10,0), Mac (10,12)]
```

Comment:
really ? In most cases that comment was because of an old limitation in *our* runtime (fixed now).